packages/community/src/llm/bedrock/base.ts

```diff
@@ -194,9 +197,59 @@ class AnthropicProvider extends Provider {
   }
 }
 
+class MetaProvider extends Provider {
+  getResultFromResponse(
+    response: Record<string, any>,
+  ): MetaNoneStreamingResponse {
+    return JSON.parse(toUtf8(response.body));
+  }
+
+  getTextFromResponse(response: Record<string, any>): string {
+    const result = this.getResultFromResponse(response);
+    return result.generation;
+  }
+
+  getTextFromStreamResponse(response: Record<string, any>): string {
+    const event: StreamEvent | undefined = response.chunk?.bytes
+      ? JSON.parse(toUtf8(response.chunk?.bytes))
+      : undefined;
+
+    if (event?.type === "content_block_delta") return event.delta.text;
```

Comment:
This is Anthropic specific the chunk from Meta looks like:

```js
{
  generation: ' hi',
  prompt_token_count: null,
  generation_token_count: 19,
  stop_reason: null
}
```

so here it should be:

`event.generation`, `StreamEvent` name could be updated to `AnthropicStreamEvent` and `MetaStreamEvent` can be created too